src/domain/error/entity/error.ts

```diff
@@ -0,0 +1,15 @@
+import type { Map } from 'immutable'
+import type { Entity } from 'domain/type/entity.type'
+
+export type Id = string
+export type Error = Entity<
+  {
+    readonly name: string
+    readonly message: string
+    readonly timeStamp: Date
```

Comment:
```suggestion
    readonly timestamp: Date
```

I think the camel case isn't justified as it is a single word

Reply:
Yep, I think it's my eyes ... 👀